example/string/read_int_test.c

```diff
@@ -6,11 +6,33 @@ int main(void)
 
   printf( "enter a string: " );
   if( !fgets( buf, BUFSIZ, stdin ) ) return 1;
-  while( buf[0] )
-    printf( "%d\n", zSInt( buf ) );
+#ifdef __WINDOWS__
```

Comment:
This example program is not platform-dependent, and so, this line is not necessary.